src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");

```

Comment:
The message should indicate what type of `Machine` is working right now. Consider adding more informative messages that specify what the Excavator is doing, for example, 'Excavator is digging'.